packages/cedar-amcharts/src/render/render.ts

```diff
@@ -75,8 +76,15 @@ export function fillInSpec(spec: any, definition: any) {
     }
   }
 
+  // Handle Legend in case.
   if (definition.legend) {
-    spec.legend.enabled = definition.legend.enable
+    const legend = definition.legend
+    if (legend.hasOwnProperty('visible')) {
```

Comment:
I don't think this `hasOwnProperty()` check is needed

Reply:
We agreed in slack that this is needed.